pkg/generate/app/environmentfile.go

```diff
@@ -0,0 +1,37 @@
+package app
+
+import (
+	"bufio"
+	"os"
+)
+
+// IsPossibleEnvironmentFile returns true if the argument is a file. I am not
+// entirely sure, why we are using this function, as this is just a wrapper,
+// we are not doing any additional validations here. This is something that I
+// am borrowing from templates file. Need feedback on this.
+func IsPossibleEnvironmentFile(value string) bool {
+	var fileexists bool = true
+	_, err := os.Stat(value)
+	if err != nil {
+		fileexists = false
+	}
+	return isFile(value) && fileexists
+}
```

Comment:
seems like it would make more sense to just move this logic inside the GetEnvironmentVars function and have GetEnvironmentVars return an error if the path is not an existing file.